e2e/rbd.go

```diff
@@ -1145,6 +1145,63 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("Validate PVC restore from vaultKMS to vaultTenantSAKMS", func() {
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 16) {
+					Skip("pvc clone is only supported from v1.16+")
+				}
+				restoreSCName := "restore-sc"
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
```

Comment:
new error logging should ideally not include the repetitive/obvious `with error`, but only `<message>: err`.

Reply:
done